src/simulator/controllers/map/camera_controller.py

```diff
@@ -0,0 +1,184 @@
+from direct.gui.OnscreenText import OnscreenText
+from direct.showbase.ShowBaseGlobal import globalClock
+from direct.showbase.DirectObject import DirectObject
+from direct.showbase.ShowBase import ShowBase
+from math import pi, sin, cos
+
+from utility.utils import exclude_from_dict
+from simulator.controllers.controller import Controller
+
+class CameraController(Controller, DirectObject):
+    __base: ShowBase
```

Comment:
Please can we not have any `__name`s? We aren't doing "defensive programming", this isn't for some security purpose, it just makes issues in the future